src/components/HomepageFeatures/index.tsx

```diff
@@ -36,11 +36,20 @@ const FeatureList: FeatureItem[] = [
       </>
     ),
   },
+  {
+    title: 'DEM',
+    Svg: require('@site/static/img/dem.svg').default,
+    description: (
+      <>
+        <strong>A digital elevation model</strong> (DEM) is a representation of the topography of the Earth or another surface in digital format, that is, by coordinates and numerical descriptions of altitude
```

Comment:
Please change from `A digital elevation model` to `Digital elevation model`.

Reply:
done digital elevation model --> Digital Elevation Model